### PR TITLE
Blog post on R&D day

### PR DESCRIPTION
**Pull request by BirgitBader**
@steffimeier @maletzke please give your feedback

@otrosien @mduesterhoeft @djablonski please give your feedback on the architecture, microservices and docker topics

Thanks

### CODE REVIEW

File: _posts/2015-09-15-rnd-day-2015.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+layout: post
+title: "R&D Day 2015 in the spotlight"
+date: "2015-09-15 08:10:00"
+icon: search-plus
+categories: tech-stories
+authors: ["Birgit"]
+---
+
+ePages developers from all company locations convened in Hamburg to spend an amazing time together on software development topics!
+It was all about interesting lectures and workshops, healthy discussions and knowledge transfer amongst different teams and departments.
```
Comment (djablonski) [inline _posts/2015-09-15-rnd-day-2015.md]: Departments? RnD is just one... ;-)

File: _posts/2015-09-15-rnd-day-2015.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+# Spotlight: ePages software architecture
+
+Team Black started the R&D Day presenting their approach on a new software architecture that should be the technical foundation for ePages in the years to come.
+
+{% image blog-rnd-architecture.jpg %}{% endimage %}
+
+The following principles drive the teams' work:
+
+* support development on the product in distributed teams
```
Comment (djablonski) [inline _posts/2015-09-15-rnd-day-2015.md]: teams distributed across epages locations

File: _posts/2015-09-15-rnd-day-2015.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+Using [node.js](https://nodejs.org/en/) and [akka](http://akka.io/), [React](http://facebook.github.io/react/) and [webpack](https://webpack.github.io/) they realised a cool and slick implementation.
+
+# Hands-on: Microservices
+
+In this workshop, the participants gained first experience on the new ePages architecture.
+They implemented a simple microservice in Java and had fun baking an [e.Pizza](https://github.com/ePages-de/rnd-microservices-handson).
+
+# Spotlight: Automating Jenkins
+
+Check out our blog post [Infrastructure as code: automating Jenkins](https://developer.epages.com/blog/2015/06/25/infrastructure-as-code.html) to find out more on this topic.
```
Comment (djablonski) [inline _posts/2015-09-15-rnd-day-2015.md]: Here you can add a teaser for the upcoming update on this topic (due ca. end of month)

**Comment by djablonski** [comment]
In general, haven't we ageed on using lower case P in epages last friday? :-D

**Comment by djablonski** [comment]
Regarding the image: I'm usually not too happy seing images of me posted publicly on the internet. But this time, it's blurry enough not to matter...

File: _posts/2015-09-15-rnd-day-2015.md (hunk 1 above)
Comment (otrosien) [inline _posts/2015-09-15-rnd-day-2015.md]: .. you forgot AM

File: _posts/2015-09-15-rnd-day-2015.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+Team Unity shared insights on the new storefront and editor.
+They explained how they plan to integrate this into the existing ePages product using the ePages [REST API](https://developer.epages.com/).
+
+Using [node.js](https://nodejs.org/en/) and [akka](http://akka.io/), [React](http://facebook.github.io/react/) and [webpack](https://webpack.github.io/) they realised a cool and slick implementation.
+
+# Hands-on: Microservices
+
+In this workshop, the participants gained first experience on the new ePages architecture.
+They implemented a simple microservice in Java and had fun baking an [e.Pizza](https://github.com/ePages-de/rnd-microservices-handson).
```
Comment (otrosien) [inline _posts/2015-09-15-rnd-day-2015.md]: suggestion: (as the participants did not really implement a microservice)
They completed hands-on exercises illuminating common microservice patterns, like synhronous and asynchronous communication and persistence - and had fun building a microservice-based pizza bakery.

Comment (maletzke) [inline _posts/2015-09-15-rnd-day-2015.md]: "Product Managements" <-- I think it should be "Product Management's"
"and app" <-- "an app"

File: _posts/2015-09-15-rnd-day-2015.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+* Themes and Affiliates
+
+The challenge was to provide an architecture which not only allows external developers to develop and submit their app easily, but also to provide an administration tool for the product management which allows them to distribute the apps to the different providers.
+
+Despite the brief time frame of the session and the complexity of the architecture, Team Red succeeded
+to present in an understandable way using several diagrams and live demos of the different App & Theme Store use cases:
+
+- developers point of view: How an app developer can submit an app.
+- Product Managements point of view: How PM is able to review an app and manage apps for the different shop types.
+- merchants point of view: How to install and uninstall and app in the merchant's administration area.
```
Comment (steffimeier) [inline _posts/2015-09-15-rnd-day-2015.md]: ,,,and uninstall an app in....

File: _posts/2015-09-15-rnd-day-2015.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+
+# Hands-on: Writing tests
+
+{% image blog-rnd-tests.jpg %}{% endimage %}
+
+In line with the Test Automation lecture, Team Ocean did a nice workshop session on writing tests.
+They dealt with:
+
+* Differences between test types
+* Why mocking is needed and how it actually workshop
```
Comment (steffimeier) [inline _posts/2015-09-15-rnd-day-2015.md]: how it actually -> how to do it

**Comment by BirgitBader** [comment]
@JessTinker please proof-read. Thank you!

File: _posts/2015-09-15-rnd-day-2015.md [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+---
+
+ePages developers from all company locations convened in Hamburg to spend an amazing time together on software development topics!
+It was all about interesting lectures and workshops, healthy discussions and knowledge transfer among different teams.
+
+Curtains up for these fantastic topics:
+
+# Spotlight: ePages software architecture
+
+Team Black started the R&D Day presenting their approach on a new software architecture that should be the technical foundation for ePages in the years to come.
```
Comment (JessTinker) [inline _posts/2015-09-15-rnd-day-2015.md]: Do you think you need to write what R&D is in full first? Probably not its self explanatory right?

File: _posts/2015-09-15-rnd-day-2015.md [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+They explained the different app types, the store currently supports:
+
+* REST apps that use the ePages [REST API](https://developer.epages.com/)
+* SOAP apps that use the ePages [SOAP API](https://developer.epages.com/soap/index.html)
+* Themes and Affiliates
+
+The challenge was to provide an architecture which not only allows external developers to develop and submit their app easily, but also to provide an administration tool for the product management which allows them to distribute the apps to the different providers.
+
+Despite the brief time frame of the session and the complexity of the architecture, Team Red succeeded
+to present in an understandable way using several diagrams and live demos of the different App & Theme Store use cases:
```
Comment (JessTinker) [inline _posts/2015-09-15-rnd-day-2015.md]: I would change to:
Team Red succeeded and presented, in an understandable way, several diagrams and live demos of the different App & Theme Store use cases:

File: _posts/2015-09-15-rnd-day-2015.md [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+
+- developers point of view: How an app developer can submit an app.
```
Comment (JessTinker) [inline _posts/2015-09-15-rnd-day-2015.md]: Perhaps remove the fullstop as you have only put a full stop at the end of the final bullet point.

File: _posts/2015-09-15-rnd-day-2015.md [hunk 10]
```diff
@@ -0,0 +1,115 @@
...
+- Product Management's point of view: How PM is able to review an app and manage apps for the different shop types.
```
Comment (JessTinker) [inline _posts/2015-09-15-rnd-day-2015.md]: Remove full stop

File: _posts/2015-09-15-rnd-day-2015.md [hunk 11]
```diff
@@ -0,0 +1,115 @@
...
+- merchants point of view: How to install and uninstall an app in the merchant's administration area.
+
+# Spotlight: New Storefront
+
+{% image blog-rnd-unity.jpg %}{% endimage %}
+
+Team Unity shared insights on the new storefront and editor.
+They explained how they plan to integrate this into the existing ePages product using the ePages [REST API](https://developer.epages.com/).
+
+Using [node.js](https://nodejs.org/en/) and [akka](http://akka.io/), [React](http://facebook.github.io/react/) and [webpack](https://webpack.github.io/) they realised a cool and slick implementation.
```
Comment (JessTinker) [inline _posts/2015-09-15-rnd-day-2015.md]: Change realised to accomplished??

File: _posts/2015-09-15-rnd-day-2015.md [hunk 12]
```diff
@@ -0,0 +1,115 @@
...
+
+# Hands-on: Microservices
+
+In this workshop, the participants gained first experience on the new ePages architecture.
```
Comment (JessTinker) [inline _posts/2015-09-15-rnd-day-2015.md]: Change to:
In this workshop, the participants gained first-hand experience of the new ePages architecture.

File: _posts/2015-09-15-rnd-day-2015.md [hunk 13]
```diff
@@ -0,0 +1,115 @@
...
+Check out our blog post [Infrastructure as code: automating Jenkins](https://developer.epages.com/blog/2015/06/25/infrastructure-as-code.html) to find out more on this topic.
+Stay tuned for an upcoming update on this topic.
+
+# Spotlight: Test Automation
+
+Ideally, new features and bug fixes are available on the live systems as soon as they are implemented and verified.
+In order to achieve that, there is a clear necessity for [automated tests](http://blog.daverooney.ca/2015/04/getting-started-with-test-driven.html).
+For each application layer, specific test automation tools and techniques exist:
+
+* The Presentation layer is what the user gets to see.
```
Comment (JessTinker) [inline _posts/2015-09-15-rnd-day-2015.md]: Remove the full stop
You also have used lower case for the first letter of the bullet point with the others?

File: _posts/2015-09-15-rnd-day-2015.md [hunk 14]
```diff
@@ -0,0 +1,115 @@
...
+* The Functionality layer represents the API of the system.
```
Comment (JessTinker) [inline _posts/2015-09-15-rnd-day-2015.md]: Remove full stop

File: _posts/2015-09-15-rnd-day-2015.md [hunk 15]
```diff
@@ -0,0 +1,114 @@
...
+
+* REST apps that use the ePages [REST API](https://developer.epages.com/)
+* SOAP apps that use the ePages [SOAP API](https://developer.epages.com/soap/index.html)
+* Themes and Affiliates
+
+The challenge was to provide an architecture which not only allows external developers to develop and submit their app easily, but also to provide an administration tool for the product management which allows them to distribute the apps to the different providers.
+
+Despite the brief time frame of the session and the complexity of the architecture, Team Red succeeded and presented, in an understandable way, several diagrams and live demos of the different App & Theme Store use cases:
+
+- developers point of view: How an app developer can submit an app
```
Comment (JessTinker) [inline _posts/2015-09-15-rnd-day-2015.md]: Sorry I think for these specific points Developers should have an uppercase D

File: _posts/2015-09-15-rnd-day-2015.md [hunk 16]
```diff
@@ -0,0 +1,114 @@
...
+- Product Management's point of view: How PM is able to review an app and manage apps for the different shop types
+- merchants point of view: How to install and uninstall an app in the merchant's administration area.
```
Comment (JessTinker) [inline _posts/2015-09-15-rnd-day-2015.md]: Sorry I think for these specific points Merchants should have an uppercase M

**Pull request merged**